src/main/java/wtf/g4s8/examples/system/Decision.java

```diff
@@ -0,0 +1,8 @@
+package wtf.g4s8.examples.system;
+
+public enum Decision {
+    NONE,
+    COMMIT,
```

Comment:
@stepanvalyavskiy what's a commit decision? RM may have only prepared or aborted decisions. I'd add javadoc to explain what does this enum mean

Reply:
`COMMIT` was for `decide to commit` and `ABORT` for `decide to abort`.
`COMMIT` renamed to `PREPARE`